terasoluna-gfw-common-libraries/terasoluna-gfw-web/src/main/java/org/terasoluna/gfw/web/exception/ExceptionLoggingFilter.java

```diff
@@ -159,7 +159,7 @@ protected void logServletException(ServletException ex,
      */
     protected void logRuntimeException(RuntimeException ex,
             ServletRequest request, ServletResponse response) {
-        exceptionLogger.log(ex);
+        exceptionLogger.error(ex);
     }

```

Comment:
@yoshikawaa

As a result of the discussion, JavaDoc comment has been deleted because information that can be inferred from the design of the class is not to be dared to write.